### DISCUSSION TITLE
Welcome to Real-Pro-Suite Discussions!

### DISCUSSION THREAD

**Discussion by waqarahmad134** [waqarahmad134/Real-Pro-Suite#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Arsel13** [comment]
What's this project about,?

**Reply by waqarahmad134** [reply]
It's a real estate website operating in Middle East.

**Comment by Arsel13** [comment]
In which language has this project been made,?

**Reply by waqarahmad134** [reply]
MERN (Mongo DB, Express JS, React JS, Node JS) Stack
